cadquery/occ_impl/assembly.py

```diff
@@ -289,3 +301,121 @@ def toJSON(
         rv.extend(toJSON(child, loc, color, tolerance))
 
     return rv
+
+
+def toFusedCAF(
+    assy: AssemblyProtocol,
+    assy_name: str,
+    glue: Optional[bool] = False,
+    tol: Optional[float] = None,
+) -> TDocStd_Document:
```

Comment:
Maybe consider updating the signature to match toCAF. The other value is used in CQ-editor